index.js

```diff
@@ -474,4 +475,32 @@ var ab2hex = function (ab) {
   return new Buffer(buffer).toString('hex')
 }
 
+// wrap around webcrypto.getRandomValues() to use high-quality PRNG, when available
+var getRandomValues = function (ab) {
+  var err, i, j, octets
+
+  if (!ab.BYTES_PER_ELEMENT) {
+    err = new Error()
+    err.name = 'TypeMisMatchError'
+    throw err
+  }
+  if (ab.length > 65536) {
+    err = new Error()
+    err.name = 'QuotaExceededError'
+    throw err
+  }
+
+  octets = crypto.randomBytes(ab.length * ab.BYTES_PER_ELEMENT)
+
+  if (ab.BYTES_PER_ELEMENT === 1) ab.set(octets)
+  else {
```

Comment:
perhaps "reasonable people" would get this, that's the minority. (-;

how about if i add this comment to the code:

     // intended to comply with https://www.w3.org/TR/WebCryptoAPI/#Crypto-method-getRandomValues
     // even though the entirety of that functionality is not required in this package!